src/molecules/rich-text-editor/useTiptapCommands.ts

```diff
@@ -0,0 +1,48 @@
+import { Editor } from '@tiptap/react'
+
+interface useTiptaponClicksProps {
+  editor: Editor
+}
+
+interface TiptapCommand {
+  name: string
+  onClick: () => void
+  icon: string
+  attributes?: Record<string, any>
+}
+
+const useTiptaponClicks = (props: useTiptaponClicksProps): TiptapCommand[] => {
```

Comment:
`useTiptaponClicks` -> `useTiptapOnClicks`? and same for type name

Reply:
Thanks for catching misspelling, should be `useTiptapCommands`